<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibrador estereoscópico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="displayLeft">
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="displayRight">
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>497</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Calibración</string>
    </property>
    <widget class="QSpinBox" name="cornersX">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>26</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="squareSize">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>86</y>
       <width>60</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>2.800000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonCalibration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>28</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibra</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>26</y>
       <width>103</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Esquinas en X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>56</y>
       <width>103</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Esquinas en Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSave">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>58</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLoad">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>88</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pairCount">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>116</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>116</y>
       <width>133</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad de pares:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cornersY">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>56</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>86</y>
       <width>133</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de casilla:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonYML">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>138</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>YML</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXML">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>118</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XML</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>497</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Cámaras</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Usar webcam integrada</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiempo entre toma de imágenes:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="displayRectification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>601</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectification</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayDepthMap">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>195</y>
      <width>352</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>DepthMap</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Parámetros del mapa de profundidad:</string>
    </property>
    <widget class="QSpinBox" name="dm_preFilterSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de prefiltrado</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tope de prefiltrado</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_preFilterCap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>63</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>63</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de ventana SAD</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_SADWindowSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Disparidad mínima</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_minDisparity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>-28</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Número de disparidades</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_numDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Umbral de textura</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_textureThr">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>32000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Indice de unicidad</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_uniqRatio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de mancha de ventana</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_spWindowSize">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rango de mancha</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dm_spRange">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIniciar_c_maras">
   <property name="text">
    <string>Iniciar cámaras</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
